ambry-store/src/main/java/com.github.ambry.store/DiskManager.java

```diff
@@ -229,6 +229,16 @@ boolean scheduleNextForCompaction(PartitionId id) {
     return store != null && compactionManager.scheduleNextForCompaction(store);
   }
 
+  /**
+   * Disable compaction on the {@link PartitionId} {@code id}.
```

Comment:
can you mention in the documentation that the disable does not take effect if the store is not up?

Reply:
This comment is no longer valid